core/__tests__/fixtures/codeConfig/manual-bootstrapped-property/config.js

```diff
@@ -0,0 +1,177 @@
+module.exports = async function getConfig() {
```

Comment:
maybe the name of this file & test case is something like `bootstrap-included-with-source` or something like that